src/gui/qgspropertyoverridebutton.cpp

```diff
@@ -119,6 +122,10 @@ void QgsPropertyOverrideButton::init( int propertyKey, const QgsProperty &proper
 
   updateFieldLists();
   updateGui();
+
+  checkCheckedWidgets( isActive() );
+  enableEnabledWidgets( isActive() );
+  visibleVisibleWidgets( isActive() );
```

Comment:
@nyalldawson this had not effect on the initial state. I don't understand why it's not correctly setup.